stable/rails/templates/ingress.yaml

```diff
@@ -1,33 +1,37 @@
 {{ if (and .Values.webserver.enabled .Values.ingress.enabled) }}
+{{- range .Values.ingress.list }}
+---
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ include "rails.name" . }}-ingress
+  name: {{ include "rails.name" $ }}-ingress-{{ .type }}
```

Comment:
In the name, we probably need to add an index of the ingress. If I define 2 public ingress configurations will it create different names for them?